src/server.rs

```diff
@@ -437,12 +444,20 @@ impl<F, H> Server<F, H> {
 
     /// Disable capturing and reporting health and metrics.
     ///
-    /// This is useful during tests.
+    /// This is useful during tests or local development.
     pub fn disable_health_and_metrics(mut self, disable_health_and_metrics: bool) -> Self {
         self.disable_health_and_metrics = disable_health_and_metrics;
         self
     }
 
+    /// Disable graceful shutdown.
+    ///
+    /// This is useful during tests or local development.
+    pub fn disable_graceful_shutdown(mut self, disable_graceful_shutdown: bool) -> Self {
```

Comment:
Do we need to take a bool as argument if it's called "disable"? Perhaps `use_graceful_shutdown` makes more sense if we want to take a bool as argument. But I guess in that case we should probably change the health and metrics as well. Anyway not something important just something I came to think about.

Reply:
Good point! I've renamed it to `with_graceful_shutdown` and `with_health_and_metrics`.